pkg/output/progress.go

```diff
@@ -47,8 +47,8 @@ func (p *progress) Start() {
 
 func (p *progress) Stop() {
 	if p.started.Swap(false) {
+		Printf("Scanning resources:    (%d)\n", p.count.Load())
```

Comment:
I think this should remain in the `render` method within `case <-p.endChan:` WDYT @moadibfr

Reply:
I'm not able to reproduce a good comportment in render()

Reply:
True, the render close can happen lately and will display inconsistent output.

You can remove useless white spaces and parenthesis there

Reply:
May be we can write `Scanned resources: %d` instead

Reply:
The reason why i keep the length of the string is because if i'm not it's display like this at the end:
`Scanned resources: 20 (18)`